src/sdk/models/templates.ts

```diff
@@ -0,0 +1,7 @@
+import { IEntityIdentifier } from './base';
+
+export interface Template extends IEntityIdentifier<string> {
```

Comment:
Templates should act on job interface. No need for the additional `Template` interface